### DISCUSSION TITLE
怎么使用命令行上传文件?

### DISCUSSION THREAD

**Discussion by wangshuai-007** [AlistGo/alist#1579] [Q&A]
我有一个linux服务器,我想要直接用命令上传文件到我的服务器上
请问alist支持这个功能吗?(最好可以支持游客上传)
我用这个命令:
`curl -X POST -d @myfilename http://alist.com/api/public/upload`

提示:
`{"code":403,"message":"Not allow upload","data":null}`

**Comment by Milky-Liu** [comment]
同问，想直接上传服务器上的文件到onedrive上

**Comment by varg1714** [comment]
同问

**Comment by naicfeng** [comment]
#2061
